fastlane/lib/fastlane/actions/get_version_number.rb

```diff
@@ -39,9 +40,14 @@ def self.run(params)
         return version_number
       end
 
-      def self.get_project!(folder)
+      def self.get_project!(xcodeproj_path_or_dir)
         require 'xcodeproj'
-        project_path = Dir.glob("#{folder}/*.xcodeproj").first
+        if File.extname(xcodeproj_path_or_dir) == ".xcodeproj"
+          project_path = xcodeproj_path_or_dir
+        else
+          project_path = Dir.glob("#{xcodeproj_path_or_dir}/*.xcodeproj").first
+        end
+
         if project_path
           return Xcodeproj::Project.open(project_path)
```

Comment:
This will raise an exception if no project it's found doesn't it?

Reply:
Looks like there're multiple reasons `Project.open(…)` can raise:

https://github.com/CocoaPods/Xcodeproj/blob/master/lib/xcodeproj/project.rb#L90-L114